webapps/trunx-docs/app/form/general/CompleteFormExample.tsx

```diff
@@ -0,0 +1,180 @@
+"use client"
+import {
+  Button,
+  Checkbox,
+  Column,
+  Columns,
+  Control,
+  Field,
+  Form,
+  Help,
+  Input,
+  Label,
+  Radio,
+  bulma,
+} from "trunx"
+import { Code, PageSection } from "@/components"
+
+export default function Example() {
+  return (
+    <PageSection title="Colors">
+      <Columns>
+        <Column size="half">
+          <Form>
+            <Field>
+              <Label>Name</Label>
+
+              <Control>
+                <Input type="text" placeholder="Text input" />
+              </Control>
+            </Field>
+
+            <Field>
+              <Label>Username</Label>
+
+              <Control>
+                <Input
```

Comment:
Can we try creating dynamic components with data of props coming in the form of json?

Reply:
yes it would be feasible in general, this is a simple example to get the user introduced with form components, I would keep it flat